ui/plugins/controller.js

```diff
@@ -146,7 +152,11 @@ treeherder.controller('PluginCtrl', [
                     // the fourth result comes from the jobLogUrl artifact
                     // exclude the json log URLs
                     $scope.job_log_urls = _.reject(results[3], {name: 'mozlog_json'});
-
+                    $scope.perf_job_url = ('http://local.treeherder.mozilla.org/perf.html#/graphs?series=[' +
+                    $scope.repoName+ ',' + $scope.job.signature + ',1]&selected=[' +
+                    $scope.repoName + ',' + $scope.job.signature + ',' + $scope.job.result_set_id +
+                        ',' + $scope.job.id + ']');
+                    console.log($scope.job.signature)
```

Comment:
Right, a job signature is not the same thing as a performance signature (it's used to identify a job, as you might expect).

I made it possible last week to query performance data for a job (see bug 1214860). E.g.:

https://treeherder.mozilla.org/api/project/mozilla-inbound/performance/data/?job_id=15891075

You'll need to look up the signatures for all those jobs seperately, of course, maybe using a similar method as we do in the compare view:

https://github.com/mozilla/treeherder/blob/master/ui/js/perf.js#L448